src/client/pages/LivePage.jsx

```diff
@@ -37,7 +37,7 @@ export default function LivePage() {
   });
 
   return (
-    <div className="flex h-screen w-screen">
+    <div className="flex h-[78vh] w-screen">
```

Comment:
i believe the h-[78vh] still shows whitespace at the bottom. i think changing this to 100vh and adding a static height to the chatbox would be a better strategy.